homeassistant/components/airvisual_pro/sensor.py

```diff
@@ -45,66 +45,74 @@ class AirVisualProMeasurementDescription(
         key="air_quality_index",
         device_class=SensorDeviceClass.AQI,
         state_class=SensorStateClass.MEASUREMENT,
-        value_fn=lambda settings, status, measurements: measurements[
+        value_fn=lambda settings, status, measurements, history: measurements[
             async_get_aqi_locale(settings)
         ],
     ),
+    AirVisualProMeasurementDescription(
+        key="outdoor_air_quality_index",
+        device_class=SensorDeviceClass.AQI,
+        state_class=SensorStateClass.MEASUREMENT,
+        value_fn=lambda settings, status, measurements, history: history.get(
+            f'Outdoor {"AQI(US)" if settings["is_aqi_usa"] else "AQI(CN)"}'
+        ),
```

Comment:
This entity description doesn't have a translation key. This would probably result in sensor.{device}_air_quality_index and sensor.{device}_air_quality_index_2. Please add a translation_key here and add something like "Outdoor air quality index" to the strings.json

Reply:
And to clarify, other entity descriptions dont have translation keys, because they get their name from the device class, Hence why this would make it confusing

Reply:
@joostlek I've updated the code to include the translation key and the relevant strings.json update, but for some reason it isn't being picked up in the UI. I've even done a fresh install just to clear any potential issues out.

Can you can a look at my changes and give me some direction on how to fix the problem? Much appreciated!

Reply:
I tried it again today and this time it's working.